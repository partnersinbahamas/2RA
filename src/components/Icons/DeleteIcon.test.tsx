import React from 'react';
import { render } from '@testing-library/react';
import DeleteIcon from './DeleteIcon';

describe('DeleteIcon', () => {
  it('renders small', () => {
    const { container } = render(<DeleteIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<DeleteIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<DeleteIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<DeleteIcon />);
    expect(container).toMatchSnapshot();
  });
});
