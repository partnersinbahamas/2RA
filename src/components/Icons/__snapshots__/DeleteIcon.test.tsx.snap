// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="4 2 24 28"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <path
      clip-rule="evenodd"
      d="M4 11C4 10.437 4.438 10 5 10H6.156C6.90946 10.0026 7.63132 10.3031 8.1641 10.8359C8.69688 11.3687 8.99736 12.0905 9 12.844V27C9 27.563 9.469 28 10 28H22.156C22.267 28.0005 22.377 27.9791 22.4796 27.9368C22.5822 27.8946 22.6755 27.8325 22.754 27.754C22.8325 27.6755 22.8946 27.5822 22.9368 27.4796C22.9791 27.377 23.0005 27.267 23 27.156V12.875C22.9989 12.4972 23.0726 12.1228 23.2167 11.7735C23.3608 11.4242 23.5725 11.1069 23.8397 10.8397C24.1069 10.5725 24.4242 10.3608 24.7735 10.2167C25.1228 10.0726 25.4972 9.99894 25.875 10H27C27.563 10 28 10.438 28 11C28 11.563 27.562 12 27 12H25.875C25.6445 12.0048 25.4247 12.0986 25.2616 12.2616C25.0986 12.4247 25.0048 12.6445 25 12.875V27.156C24.9974 27.9095 24.6969 28.6313 24.1641 29.1641C23.6313 29.6969 22.9095 29.9974 22.156 30H10C9.20435 30 8.44129 29.6839 7.87868 29.1213C7.31607 28.5587 7 27.7956 7 27V12.844C7.00053 12.733 6.97906 12.623 6.93683 12.5204C6.89461 12.4178 6.83246 12.3245 6.75398 12.246C6.6755 12.1675 6.58225 12.1054 6.47961 12.0632C6.37697 12.0209 6.26698 11.9995 6.156 12H5C4.437 12 4 11.562 4 11ZM20 2C20.563 2 21 2.438 21 3V6H27C27.563 6 28 6.438 28 7C28 7.563 27.562 8 27 8H5C4.437 8 4 7.562 4 7C4 6.437 4.438 6 5 6H11V3C11 2.437 11.469 2 12 2H20ZM19 4H13V6H19V4ZM11 25V14C11 13.437 11.469 13 12 13C12.563 13 13 13.438 13 14V25C13 25.563 12.562 26 12 26C11.469 26 11 25.562 11 25ZM15 25V14C15 13.437 15.438 13 16 13C16.563 13 17 13.438 17 14V25C17 25.563 16.562 26 16 26C15.437 26 15 25.562 15 25ZM19 25V14C19 13.437 19.469 13 20 13C20.563 13 21 13.438 21 14V25C21 25.563 20.562 26 20 26C19.469 26 19 25.562 19 25Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`DeleteIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="4 2 24 28"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <path
      clip-rule="evenodd"
      d="M4 11C4 10.437 4.438 10 5 10H6.156C6.90946 10.0026 7.63132 10.3031 8.1641 10.8359C8.69688 11.3687 8.99736 12.0905 9 12.844V27C9 27.563 9.469 28 10 28H22.156C22.267 28.0005 22.377 27.9791 22.4796 27.9368C22.5822 27.8946 22.6755 27.8325 22.754 27.754C22.8325 27.6755 22.8946 27.5822 22.9368 27.4796C22.9791 27.377 23.0005 27.267 23 27.156V12.875C22.9989 12.4972 23.0726 12.1228 23.2167 11.7735C23.3608 11.4242 23.5725 11.1069 23.8397 10.8397C24.1069 10.5725 24.4242 10.3608 24.7735 10.2167C25.1228 10.0726 25.4972 9.99894 25.875 10H27C27.563 10 28 10.438 28 11C28 11.563 27.562 12 27 12H25.875C25.6445 12.0048 25.4247 12.0986 25.2616 12.2616C25.0986 12.4247 25.0048 12.6445 25 12.875V27.156C24.9974 27.9095 24.6969 28.6313 24.1641 29.1641C23.6313 29.6969 22.9095 29.9974 22.156 30H10C9.20435 30 8.44129 29.6839 7.87868 29.1213C7.31607 28.5587 7 27.7956 7 27V12.844C7.00053 12.733 6.97906 12.623 6.93683 12.5204C6.89461 12.4178 6.83246 12.3245 6.75398 12.246C6.6755 12.1675 6.58225 12.1054 6.47961 12.0632C6.37697 12.0209 6.26698 11.9995 6.156 12H5C4.437 12 4 11.562 4 11ZM20 2C20.563 2 21 2.438 21 3V6H27C27.563 6 28 6.438 28 7C28 7.563 27.562 8 27 8H5C4.437 8 4 7.562 4 7C4 6.437 4.438 6 5 6H11V3C11 2.437 11.469 2 12 2H20ZM19 4H13V6H19V4ZM11 25V14C11 13.437 11.469 13 12 13C12.563 13 13 13.438 13 14V25C13 25.563 12.562 26 12 26C11.469 26 11 25.562 11 25ZM15 25V14C15 13.437 15.438 13 16 13C16.563 13 17 13.438 17 14V25C17 25.563 16.562 26 16 26C15.437 26 15 25.562 15 25ZM19 25V14C19 13.437 19.469 13 20 13C20.563 13 21 13.438 21 14V25C21 25.563 20.562 26 20 26C19.469 26 19 25.562 19 25Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`DeleteIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="3 1.8 18 20"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <path
      clip-rule="evenodd"
      d="M3 8C3 7.437 3.438 7 4 7H5C6.094 7 7 7.906 7 9V20H17V9C17 7.906 17.906 7 19 7H20C20.563 7 21 7.438 21 8C21 8.563 20.562 9 20 9H19V20C19 21.094 18.094 22 17 22H7C5.906 22 5 21.094 5 20V9H4C3.437 9 3 8.562 3 8ZM3 5C3 4.437 3.438 4 4 4H8V3C8 2.437 8.438 2 9 2H15C15.2639 2.00437 15.5157 2.11114 15.7023 2.29773C15.8889 2.48433 15.9956 2.73615 16 3V4H20C20.563 4 21 4.438 21 5C21 5.563 20.562 6 20 6H4C3.437 6 3 5.562 3 5ZM9 17V10C9 9.469 9.469 9 10 9C10.563 9 11 9.469 11 10V17C11 17.531 10.562 18 10 18C9.437 18 9 17.531 9 17ZM13 17V10C13 9.469 13.469 9 14 9C14.563 9 15 9.469 15 10V17C15 17.531 14.562 18 14 18C13.437 18 13 17.531 13 17Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`DeleteIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="2 2 16 16"
    xmlns="http://www.w3.org/2000/svg"
    xmlns:xlink="http://www.w3.org/1999/xlink"
  >
    <path
      clip-rule="evenodd"
      d="M2 8C2 7.437 2.438 7 3 7H4C5.094 7 6 8 6 9V16H14.031V9C14.031 8 14.937 7 16 7H17C17.563 7 18 7.438 18 8C18 8.563 17.562 9 17 9H16.031V16.188C16.031 17.25 15.064 18 14.031 18H6C4.936 18 4 17.25 4 16.187V9H3C2.437 9 2 8.562 2 8ZM2 5C2 4.437 2.438 4 3 4H7V3C7 2.437 7.438 2 8 2H12C12.25 2 12.531 2.094 12.719 2.281C12.8116 2.37592 12.8842 2.48847 12.9325 2.61198C12.9807 2.73549 13.0037 2.86744 13 3V4H17C17.563 4 18 4.438 18 5C18 5.563 17.562 6 17 6H3C2.437 6 2 5.562 2 5ZM7 14V9C7 8.437 7.438 8 8 8C8.563 8 9 8.438 9 9V14C9 14.531 8.562 15 8 15C7.75 15 7.469 14.875 7.281 14.687C7.09996 14.5043 6.99888 14.2572 7 14ZM11 14V9C11 8.437 11.438 8 12 8C12.563 8 13 8.438 13 9V14C13 14.531 12.562 15 12 15C11.75 15 11.469 14.875 11.281 14.687C11.1 14.5043 10.9989 14.2572 11 14V14Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
